grafana-plugin/src/models/user_group/user_group.ts

```diff
@@ -46,6 +46,21 @@ export class UserGroupStore extends BaseStore {
     });
   }
 
+  @action.bound
+  async fetchItemById(id: string) {
+    const response: UserGroup[] = (await makeRequest(`${this.path}`, {
+      params: { id },
+    })) as UserGroup[];
+
+    runInAction(() => {
+      this.items = {
+        ...this.items,
+        // backend returns an array
+        [id]: Array.isArray(response) ? (response.length > 0 ? response[0] : undefined) : response,
```

Comment:
For some reason, backend returns an array when fetching an individual response instead of an object. Added this so that if backend will decide to change it in the near future, it won't cause a npe on the frontend